test/kubemark/start-kubemark.sh

```diff
@@ -27,6 +27,11 @@ source "${KUBE_ROOT}/test/kubemark/skeleton/util.sh"
 source "${KUBE_ROOT}/test/kubemark/cloud-provider-config.sh"
 source "${KUBE_ROOT}/test/kubemark/${CLOUD_PROVIDER}/util.sh"
 source "${KUBE_ROOT}/cluster/kubemark/${CLOUD_PROVIDER}/config-default.sh"
+
+if [ -f "${CLOUD_PROVIDER}/startup.sh" ] ; then
```

Comment:
Please change this to absolute path to be safe.
Also, I think it's safer to use [[ ]] instead of [ ] in general in bash.